### PR TITLE
Fixing URLs and Repositories, deleting some unavailable game

### PR DESCRIPTION
**Pull request by pehsa**
Fixing broke URL links
Fixing broke Repository links
Deleted some unrelevant info: Replicant, Secret Maryo Chronicles
Deleted some game, which was removed from the net: PKMN-FX, One Way to Go, Terraria (Reverse Engineer project), Uncharted Waters 2
Removed Play Freeciv and added as Freeciv-Web. They moved their page, and repository

### CODE REVIEW

**Comment by pehsa** [comment]
I fixed the duplicated repo tag for the FreeDroidRPG, but the test shows the old status

**Comment by pehsa** [comment]
I give up!

File: games/c.yaml [hunk 1]
```diff
@@ -574,6 +572,7 @@
   type: clone
   updated: 2013-05-29
   url: http://crimson.seul.org/
+  repo: http://crimson.seul.org/files/crimson-0.5.3.tar.gz
```
Comment (cxong) [inline games/c.yaml]: `repo` should be used to point to their source control, not a direct download

File: games/f.yaml [hunk 2]
```diff
@@ -347,9 +361,9 @@
   development: complete
   originals:
   - Paradroid
-  repo: https://gitlab.com/freedroid/freedroid-src
+  repo: https://sourceforge.net/projects/freedroid/files/freedroidClassic/FreedroidClassic-1.0.2/freedroid-1.0.2.tar.gz/download
```
Comment (cxong) [inline games/f.yaml]: why change this? https://gitlab.com/freedroid/freedroid-src looks fine

File: games/f.yaml [hunk 3]
```diff
@@ -347,9 +361,9 @@
...
   type: remake
-  url: http://www.freedroid.org/
+  url: http://www.freedroid.org/download/
```
Comment (cxong) [inline games/f.yaml]: I think http://www.freedroid.org works better

File: games/f.yaml [hunk 4]
```diff
@@ -434,10 +448,9 @@
   development: active
   originals:
   - Warlords II
-  repo: https://gitlab.com/freedroid/freedroid-src
   type: remake
   updated: 2015-04-06
-  url: https://sourceforge.net/p/freelords/wiki/Home/
+  repo: https://sourceforge.net/p/freelords/wiki/Home/
```
Comment (cxong) [inline games/f.yaml]: this looks more like a url; yes it's on sourceforge but it doesn't show source control information on this page
for repo https://sourceforge.net/projects/freelords/ is fine

File: games/h.yaml [hunk 5]
```diff
@@ -288,5 +289,6 @@
   status: playable
   type: remake
   updated: 2012-05-28
-  url: http://sourceforge.net/p/hurrican/code/HEAD/tree/trunk/
+  url: https://sourceforge.net/projects/hurrican/
+  repo: http://sourceforge.net/p/hurrican/code/HEAD/tree/trunk/
```
Comment (cxong) [inline games/h.yaml]: just repo pointing to sourceforge is fine

File: games/i.yaml [hunk 6]
```diff
@@ -63,10 +63,10 @@
   content: open
   originals:
   - Ace of Spades
-  repo: https://github.com/iamgreaser/iceball
+  repo: https://github.com/arsdragonfly/iceball
```
Comment (cxong) [inline games/i.yaml]: I'm not familiar with this project so what's the story here? Is this fork hostile or officially sanctioned?

File: games/o.yaml [hunk 7]
```diff
@@ -52,20 +52,6 @@
   updated: 2015-04-04
   url: http://davidgow.net/keen/omnispeak.html
 
-- name: One Way To Go
```
Comment (cxong) [inline games/o.yaml]: looks like the project simply moved to https://gordebak.itch.io/onewaytogo

File: games/s.yaml [hunk 8]
```diff
@@ -142,7 +141,6 @@
   url: https://secretchronicles.org
 
 - name: Secret Maryo Chronicles
-  info: 'SMC is no longer under active development. However, there is now a new game based on SMC: https://github.com/secretchronicles/TSC'
```
Comment (cxong) [inline games/s.yaml]: why remove this info?

File: games/r.yaml [hunk 9]
```diff
@@ -145,7 +145,6 @@
   url: http://cyxdown.free.fr/reminiscence/
 
 - name: replicant
-  info: This version is no longer being developed, please follow https://github.com/madmoose/scummvm/tree/bladerunner/engines/bladerunner for new development.
```
Comment (cxong) [inline games/r.yaml]: why remove this info?

File: games/s.yaml [hunk 10]
```diff
@@ -498,18 +497,6 @@
   updated: 2018-08-05
   url: https://smcameron.github.io/space-nerds-in-space/
 
-- name: Space Station 14
```
Comment (cxong) [inline games/s.yaml]: This project seems to be there still, why remove?

**Comment by cxong** [comment]
Great effort, but please resubmit with only a few changes at a time; this will make it easier to review. Thanks!

**Comment by pehsa** [comment]
I will remade this file by file, and I will check your suggestions

**Pull request closed without merging**